src/Components/TaskForm.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { FC } from 'react'
+
+import {
+    Box,
+    Button,
+    Container,
+    InputLabel,
+} from '@material-ui/core';
+
+import { Formik, Form, Field, ErrorMessage } from 'formik'
+
+import schema from './Validation/schema';
+
+interface Props {
```

Comment:
I suggest moving this interface to new file `./typings.ts`

Reply:
Resolved on c9df04d6f064280d8ad37a670f759a4071b2d635